README.md

```diff
@@ -33,20 +33,21 @@ are typically self-explanatory.
-        c. Enter the command `pwd` into your terminal then click enter. Your "working directory" should
-           be in the "programming" folder, and the path response you get from the pwd command should end in "programming"
-           (i.e. user/Documents/programming)
-        d. Create a folder called "python-work" by entering the command `mkdir python-work`
-        e. If you want to have these lessons stored locally on your computer and/or your mentor advises you to do so, type 
-           `git clone https://github.com/projectinnovatenewark/csx.git` into your command terminal.
-        f. Download the following VS Code extensions- Python and Highlight. You can find the extensions tab in VS
-           Code by clicking "Shift + Ctrl + X" (windows) or "Shift + Command + X" (mac). Once there, search for those exact extensions and download the first result for each of them. Once you are done downloading those extensions, return back to the main "Explorer" page where you will be viewing your folders/files by clicking "Shift + Ctrl + E" (windows) or "Shift + Command + E" (mac). There are icons on the left panel of your VS Code to navigate through these different pages as well. You will see how these extensions are to be used at the bottom of this README
+   - To open a terminal in VS Code click "Shift + Ctrl + ~"
+
+   - If at any point in your terminal entries the beginning line of your terminal starts with ">>", ">>>", or is seemingly unable to enter your commands, click "Ctrl + C". That will quit any process running in your terminal.Sidebar- three arrows probably means you start a Python shell in your terminal! Another way to quit this type of process would be to type "quit()" and then enter.
+
+   - Enter the command `pwd` into your terminal then click enter. Your "working directory" should be in the "programming" folder, and the path response you get from the pwd command should end in "programming" (i.e. user/Documents/programming)
+
+   - Create a folder called "python-work" by entering the command `mkdir python-work`
+
+   - If you want to have these lessons stored locally on your computer and/or your mentor advises you to do so, type `git clone https://github.com/projectinnovatenewark/csx.git` into your command terminal.
+
+   - Download the following VS Code extensions- Python and Highlight. You can find the extensions tab in VS Code by clicking "Shift + Ctrl + X" (windows) or "Shift + Command + X" (mac). Once there, search for those exact extensions and download the first result for each of them. Once you are done downloading those extensions, return back to the main "Explorer" page where you will be viewing your folders/files by clicking "Shift + Ctrl + E" (windows) or "Shift + Command + E" (mac). There are icons on the left panel of your VS Code to navigate through these different pages as well. 
```

Comment:
Christian also suggested here to have may be something like - 
Download the following VS Code extensions - Python and Highlight. You can find the extensions tab in VS Code by clicking "Shift + Ctrl + X" (windows) or "Shift + Command + X" (mac). Once there, search for those exact extensions. The author of the "Python" extension is Microsoft and the author of "Highlight" is Fabio Spampinato. Once you are done downloading those extensions, return back to the main "Explorer" page where you will be viewing your folders/files by clicking "Shift + Ctrl + E" (windows) or "Shift + Command + E" (mac). There are icons on the left panel of your VS Code to navigate through these different pages as well.

Note: You should also have the VS Code `settings.json` file that comes with the csx repository copied and pasted into your own `settings.json` file. To access your own `settings.json` file click "Shift + Control (Command for Mac) + P" and type in "Open Settings (JSON)" and click on the option of the same name. That file should be scrapped and replaced entirely with the `settings.json` file that comes with the csx repository that is stored in the folder called ".vscode".